_data/versions.yaml

```diff
@@ -0,0 +1,7 @@
+quarkus:
+  version: 0.18.0
+  announce: /blog/quarkus-0-18-released/
+graalvm: 19.0.2+
+jdk: "8 to 11"
```

Comment:
It's a detail but I would say `8 or 11`.

This makes it clear that we only test and support the LTS versions.